### PR TITLE
[stable/memcached] Add extraArgs to values

### PR DESCRIPTION
**Pull request by c0dearm**
#### What this PR does / why we need it:

Adds the possibility to include additional arguments to the memcached command. This is sometimes required by users, for example to set a different value for the memcached value maximum size.

#### Which issue this PR fixes
  - fixes #9587

### CODE REVIEW

File: stable/memcached/templates/statefulset.yaml [hunk 1]
```diff
@@ -50,14 +50,21 @@ spec:
...
-        - -m {{ .Values.memcached.maxItemMemory  }}
+        - -m {{ .Values.memcached.maxItemMemory }}
         {{- if .Values.memcached.extendedOptions }}
         - -o
         - {{ .Values.memcached.extendedOptions }}
         {{- end }}
         {{- if .Values.memcached.verbosity }}
         - -{{ .Values.memcached.verbosity }}
         {{- end }}
+        {{- range $key, $value := .Values.memcached.extraArgs }}
```
Comment (olemarkus) [inline stable/memcached/templates/statefulset.yaml]: Can you change this to use `with`? See an example on line 20.

File: stable/memcached/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: memcached
-version: 2.7.1
+version: 2.7.2
```
Comment (olemarkus) [inline stable/memcached/Chart.yaml]: As this adds a new feature, the minor version should be bumped.

**Review by olemarkus — CHANGES_REQUESTED** [review]
Thanks for this PR!

File: stable/memcached/Chart.yaml (hunk 2 above)
Comment (c0dearm) [reply stable/memcached/Chart.yaml]: Fixed to version `2.8.1`

File: stable/memcached/templates/statefulset.yaml (hunk 1 above)
Comment (c0dearm) [reply stable/memcached/templates/statefulset.yaml]: Using `with` statement now. The `extraArgs` value had to be changed from `map` to `list`.

**Comment by olemarkus** [comment]
/lgtm

Thanks for this contribution

**Pull request merged**